skimage/measure/_marching_cubes_lewiner.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 from . import _marching_cubes_lewiner_luts as mcluts
-from . import _marching_cubes_lewiner_cy
+from . import _marching_cubes_lewiner_cy  # type: ignore
```

Comment:
Why do Cython modules need to be ignored?

Reply:
See https://github.com/python/mypy/issues/8575, mypy just does not read `.pyx` files.

Reply:
Also reading https://github.com/python/mypy/issues/7542 now.